roles/ceph-mon/tasks/create_mds_filesystems.yml

```diff
@@ -10,21 +10,31 @@
     - cephfs_metadata
   changed_when: false
 
+- name: check if ceph filesystem already exists
+  command: ceph --cluster {{ cluster }} fs get {{ cephfs }}
+  register: check_existing_cephfs
+  changed_when: false
+  failed_when: false
```

Comment:
Please remove `failed_when: false` and we will be good. If something goes wrong we want to fail.